frontend/src/components/modals/menu/SaveProjectModal.vue

```diff
@@ -19,11 +19,22 @@
     export default defineComponent({
         components: { AlgoModal },
 
+        data() {
+            return {
+                tempTitle: "",
```

Comment:
Nie podoba mi się prefix "temp" zwłaszcza jeżeli to nie jest zmienna dostępna tylko w obrębie pojedynczej funkcji. Proponowałbym nazwę `newTitle` lub `titleModel` lub coś podobnego.